test/position-managers/Execute.t.sol

```diff
@@ -119,7 +121,66 @@ contract ExecuteTest is Test, PosmTestSetup, LiquidityFuzzers {
     }
 
     // rebalance: burn and mint
-    function test_execute_rebalance() public {}
+    function test_execute_rebalance_perfect() public {
+        uint256 initialLiquidity = 100e18;
+
+        // mint a position on range [-300, 300]
+        BalanceDelta delta = mint(range, initialLiquidity, address(this), ZERO_BYTES);
+        uint256 tokenId = lpm.nextTokenId() - 1;
```

Comment:
bleh.. this is a case for mint to return the tokenId

Reply:
can you add a comment above that this tokenId is referencing the just minted token? or define it before the mint.. just bit confusing subtracting 1